tests/plugins/clipboard_figure.test.ts

```diff
@@ -153,3 +155,143 @@ describe("Clipboard for figures", () => {
     });
   });
 });
+
+// --------------------------------------------
+
+describe("Clipboard for image figures", () => {
+  let model: Model;
+  let sheetId: UID;
+  let imageId: UID;
+
+  beforeEach(async () => {
+    model = new Model();
+    sheetId = model.getters.getActiveSheetId();
+    imageId = model.uuidGenerator.uuidv4();
+    createImage(model, { figureId: imageId, definition: { sheetId: sheetId } });
```

Comment:
You don't need the argument `definition: { sheetId: sheetId }`, the helper already have the active sheet id as default value.